Package.swift

```diff
@@ -18,14 +18,15 @@ let package = Package(
     dependencies: [
         // Dependencies declare other packages that this package depends on.
         // .package(url: /* package url */, from: "1.0.0"),
-        .package(url: "https://github.com/attaswift/BigInt.git", from: "5.3.0")
+        .package(url: "https://github.com/attaswift/BigInt.git", from: "5.3.0"),
+        .package(url: "https://github.com/krzyzanowskim/CryptoSwift.git", .upToNextMajor(from: "1.6.0"))
```

Comment:
This causes an issue to any app that integrates Wallet Connect v1 whose last version still relies on CryptoSwift 1.5.x. Moving away to [keccak-tiny](https://github.com/coruus/keccak-tiny) solves the issue altogether but if for some reason it needs to be kept it would be better to downgrade to 1.5.1.

Reply:
Thanks for noting this. For now I'll downgrade to 1.5.1 and will move to keccak-tiny, once crypto-cpp integration is finished.